src/main/java/de/tum/in/www1/artemis/repository/ExamRepository.java

```diff
@@ -132,9 +132,19 @@ where TYPE(ex) = QuizExercise
     // The problem is that 2000 student Exams with 10 exercises and 2000 existing participations would load 2000*10*2000 = 40 mio objects
     // @EntityGraph(type = LOAD, attributePaths = { "studentExams", "studentExams.exercises", "studentExams.exercises.participations" })
 
-    @Query("select distinct exam from Exam exam left join fetch exam.studentExams studentExams left join fetch exam.exerciseGroups exerciseGroups left join fetch exerciseGroups.exercises where (exam.id = :#{#examId})")
+    @Query("""
+              select distinct exam
+              from Exam exam
+                  left join fetch exam.studentExams studentExams
+                  left join fetch exam.exerciseGroups exerciseGroups
+                  left join fetch exerciseGroups.exercises
+              where (exam.id = :#{#examId})
+            """)
     Exam findOneWithEagerExercisesGroupsAndStudentExams(@Param("examId") long examId);
 
+    @Query("select distinct exam from Exam exam left join fetch exam.studentExams studentExams left join fetch studentExams.exercises where (exam.id = :#{#examId})")
```

Comment:
💡 It can be nicer if you can format the JPQL by using multi-line string literals `"""`.